alfred-telemetry-platform/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
     testImplementation "org.hamcrest:hamcrest-all:${hamcrestVersion}"
     testImplementation "org.awaitility:awaitility:${awaitilityVersion}"
     testImplementation "org.mockito:mockito-inline:${mockitoVersion}"
+    testImplementation "net.bytebuddy:byte-buddy:${byteBuddyVersion}"
```

Comment:
Why was this dependency needed ?

Reply:
Bart Van Lommel
  [09:43](https://xenitengineering.slack.com/archives/D051JEWPBFE/p1701161026622409)
because it was pulled out of another "standard" Java lib (Mockito I believe) and without it the tests won't work